be/src/util/doris_metrics.h

```diff
@@ -137,6 +137,20 @@ class DorisMetrics {
     // permits required by the compaction task which is waitting for permits
     IntGauge* compaction_waitting_permits;
 
+    UIntGauge* tablet_version_num_distribution_num;
+    UIntGauge* tablet_version_num_distribution_min;
+    UIntGauge* tablet_version_num_distribution_max;
+    UIntGauge* tablet_version_num_distribution_sum;
+    DoubleGauge* tablet_version_num_distribution_median;
+    DoubleGauge* tablet_version_num_distribution_average;
+    DoubleGauge* tablet_version_num_distribution_standard_deviation;
+    DoubleGauge* tablet_version_num_distribution_p50;
+    DoubleGauge* tablet_version_num_distribution_p75;
+    DoubleGauge* tablet_version_num_distribution_p90;
+    DoubleGauge* tablet_version_num_distribution_p95;
+    DoubleGauge* tablet_version_num_distribution_p99;
+
```

Comment:
Use HistogramMetric directly, its output is standard now.

Reply:
> Use HistogramMetric directly, its output is standard now.

OK,  I see it too. PR #5671 has been merged.